### DISCUSSION TITLE
Title: DAI 2024-2025 - My picocli project - Alexandre Delétraz

### DISCUSSION THREAD

**Discussion by Nekoth93** [heig-vd-dai-course/heig-vd-dai-course#401] [Show and tell]
https://github.com/Nekoth93/DAI_practical04

**Comment by ludelafo** [comment]
Thank you!

It seems a gitignore file is missing at the root level of the repository. The `target` directory should not be committed to the repository. The rest seems OK to me.

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
